Content/posts/improvements-to-package-search.md

```diff
@@ -0,0 +1,40 @@
+---
+date: 2021-12-15 12:00
+title: Improvements to package search
+description:
+---
+
+Recently, we’ve been focusing our attention on improving search here on the Swift Package Index, and it’s time to let you all know what we’ve implemented!
+
+### Search Results
+
+First up, we improved the information you’ll see when you see search results or any list of packages:
+
+<picture class="shadow">
+  <source srcset="/images/search-results~dark.png" media="(prefers-color-scheme: dark)">
+  <img src="/images/search-results~light.png" alt="Search results that include the number of stars a package has and when the repository last had maintenance activity, in addition to the standard metadata.">
+</picture>
+
+All package lists across the whole site now include the number of stars and how recently a package has had maintenance activity so you can start making some decisions before you even open a package page. You can see examples of this in [search results](https://swiftpackageindex.com/search?query=http), [keyword results](https://swiftpackageindex.com/keywords/rxswift), and [owner/author pages](https://swiftpackageindex.com/apple).
+
+**Note:** We update the date of the last “maintenance activity” when there is a commit to the default branch or when someone closes or merges an issue or pull request.
+
+### Search Filters
+
+We’ve also added filters so you can refine any set of search results. The best way to explain this feature is with some examples:
+
+- [Packages matching “http” that are compatible with iOS and Linux](https://swiftpackageindex.com/search?query=http+platform%3Aios%2Clinux).
+- [Packages matching “URLSession” that have had maintenance activity within the last three months](https://swiftpackageindex.com/search?query=URLSession+last_activity%3A%3E2021-09-16).
```

Comment:
If someone reads this in a month it'll be "within the last four months" :) Not sure how to fix this or if we should even bother. Maybe we can add `within the last three months at time of writing` to signal awareness.

Reply:
Yea I did consider that, I'll update it.

Reply:
Actually, it's awkward given it's in a list. I think people will figure it out.